### PR TITLE
Improve source verification tooling

### PR DESCRIPTION
**Pull request by ironcladlou**
* Verify generated go-bindata output
* Vendor go-bindata
* Apply gofmt more universally
* Clean up Makefile

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -9,8 +9,34 @@ required = [
...
   "k8s.io/gengo/args",
+  "github.com/kevinburke/go-bindata",
 ]
 
+[prune]
+  go-tests = true
+  non-go = true
+  unused-packages = true
+
+[[prune.project]]
```
Comment (pravisankar) [inline Gopkg.toml]: Don't this need to be indent under `[prune]` section? (https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md#prune)

File: Gopkg.toml [hunk 2]
```diff
@@ -9,8 +9,34 @@ required = [
...
+  name = "k8s.io/code-generator"
+  non-go = false
+  unused-packages = false
+
+[[prune.project]]
+  name = "github.com/kevinburke/go-bindata"
```
Comment (pravisankar) [inline Gopkg.toml]: For cluster-ingress-operator repo, we could change "github.com/kevinburke/go-bindata" dep from `[[constraint]]` to `[[prune.project]]`

**Comment by imcsk8** [comment]
LGTM

File: Gopkg.toml (hunk 1 above)
Comment (ironcladlou) [reply Gopkg.toml]: Indentation isn't significant in TOML.

File: Gopkg.toml (hunk 2 above)
Comment (ironcladlou) [reply Gopkg.toml]: Yeah, ingress could use some cleaning up with pruning rules.

**Comment by ironcladlou** [comment]
>Any reason not to make "all" a phony target? Otherwise looks fine.

I went ahead and fixed it

**Comment by pravisankar** [comment]
We should also make corresponding `.PHONY: all` change in cluster-ingress-operator Makefile.

**Pull request merged**